packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -800,15 +811,15 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
         .map((HitTestEntry candidate) => candidate.target)
         .whereType<RenderObject>()
         .first;
-      final Element? innerTargetElement = collectAllElementsFrom(
-        binding.renderViewElement!,
-        skipOffstage: true,
-      ).cast<Element?>().lastWhere(
```

Comment:
What was wrong with casting this to `Element?` and then using the real `lastWhere` with on `orElse: () => null`?

Reply:
It somehow throws strange null assertions, saying `null can not be cast to Element`. I guess I'll write a manual traversal then.